src/Flag.js

```diff
@@ -7,28 +7,36 @@ import Box from './Box'
 
 const capitalize = str => str.charAt(0).toUpperCase() + str.slice(1)
 
-const darkBorderColor = props => {
+const shadowColor = props => {
   const darkColor = theme(`colors.dark${capitalize(props.color)}`)(props)
+
+  if (!darkColor) {
+    return {
+      backgroundImage: `
+        linear-gradient(45deg, transparent 50%, rgba(0, 0, 0, 0.5) 50%),
+        linear-gradient(45deg, transparent 50%, ${props.color} 50%)
+      `
+    }
+  }
+
   return {
-    borderTopColor: darkColor,
-    borderRightColor: darkColor
+    backgroundImage: `linear-gradient(45deg, transparent 50%, ${darkColor} 50%)`
   }
 }
```

Comment:
Could these two `return` statements consolidate into one `return` statement?
```
return !darkColor
  ? { ... }
  : { ... }
```
Or the ternary goes inside the returned `backgroundImage` object key, as it seems the ternary conditions return the same `backgroundImage` object key?
```
return {
  backgroundImage: !darkColor ? `...` : `...`
   }
```

Which one would you like better, if you decide to consider consolidating?

Reply:
I updated it to
```
  return {
    backgroundImage: !darkColor
      ? `
        linear-gradient(45deg, transparent 50%, rgba(0, 0, 0, 0.5) 50%),
        linear-gradient(45deg, transparent 50%, ${props.color} 50%)
      `
      : `linear-gradient(45deg, transparent 50%, ${darkColor} 50%)`
  }
```